R/options.R

```diff
@@ -44,4 +44,4 @@ m_viewer_spec <- function(
 
   class(viewer_spec) <- "ViewerSpec"
   viewer_spec
-}
+}
```

Comment:
On, I'm sorry about that maybe newline is needed here.
And the test file name should be changed at the same time (test-view-options.R -> test-options.R)